libflux/flux-core/src/semantic/types.rs

```diff
@@ -120,25 +127,29 @@ impl PolyType {
             .collect::<Vec<_>>()
             .join(" + ")
     }
+    #[allow(missing_docs)]
     pub fn normal(&self) -> PolyType {
         self.clone()
             .fresh(&mut Fresher::from(0), &mut TvarMap::new())
     }
 }
 
+#[allow(missing_docs)]
 pub fn union<T: PartialEq>(mut vars: Vec<T>, mut with: Vec<T>) -> Vec<T> {
     with.retain(|tv| !vars.contains(tv));
     vars.append(&mut with);
     vars
 }
 
+#[allow(missing_docs)]
 pub fn minus<T: PartialEq>(vars: &[T], mut from: Vec<T>) -> Vec<T> {
     from.retain(|tv| !vars.contains(tv));
     from
 }
 
-#[derive(Debug, PartialEq)]
 /// Errors that can be returned during type inference.
+#[derive(Debug, PartialEq)]
+#[allow(missing_docs)]
```

Comment:
This has a doc no? `Errors that can be returned during type inference.` I would add that the intention is that these error messages are read by end users and so should keep that context in mind.